packages/kbn-server-http-tools/src/types.ts

```diff
@@ -20,6 +20,7 @@ export interface IHttpConfig {
   ssl: ISslConfig;
   shutdownTimeout: Duration;
   restrictInternalApis: boolean;
+  oas: IOasConfig;
```

Comment:
type check fails.
```suggestion
oas: IOasConfig | undefined;
```
or
```suggestion
oas?: IOasConfig;
```